application/config/development.py

```diff
@@ -2,8 +2,8 @@
 
 ADMIN_HOST = 'https://admin-beta.development.performance.service.gov.uk'
 
-BACKDROP_HOST = 'http://www.development.performance.service.gov.uk'
-STAGECRAFT_HOST = 'http://stagecraft.development.performance.service.gov.uk'
+BACKDROP_HOST = 'http://localhost:3039'
+STAGECRAFT_HOST = 'http://localhost:3204'
```

Comment:
can't these still be *.dev.gov.uk as the hosts file on the machine should loopback them?